pkg/router/routes/web/static/html/partials/modals.html

```diff
@@ -0,0 +1,110 @@
+{{define "config-modal"}}
+<div class="modal fade dark-mode" id="configModal" tabindex="-1" aria-labelledby="configModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered">
+        <div class="modal-content bg-dark text-light">
+            <div class="modal-header">
+                <h5 class="modal-title text-light" id="configModalLabel">Edit Configuration</h5>
+                <button type="button" class="btn-close btn-close-white" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <form action="/api/config/update" method="POST">
+                    <div class="mb-3">
+                        <label for="portInput" class="form-label text-light">Port</label>
+                        <input type="number" class="form-control bg-white text-dark" id="portInput" name="port" min="1" max="65535" value="{{.Config.Port}}" required>
+                    </div>
+                    <div class="mb-3">
+                        <label for="intervalInput" class="form-label text-light">Interval (in seconds)</label>
+                        <input type="number" class="form-control bg-white text-dark" id="intervalInput" name="interval" min="10" value="{{.Config.Interval}}" required>
+                    </div>
+                    <button type="submit" class="btn btn-primary">Save</button>
+                </form>
+            </div>
+        </div>
+    </div>
+</div>
+{{end}}
+
+{{define "add-modal"}}
+<div class="modal fade dark-mode" id="addModal" tabindex="-1" aria-labelledby="addModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered">
+        <div class="modal-content bg-dark text-light">
+            <div class="modal-header">
+                <h5 class="modal-title" id="addModalLabel">Add Job</h5>
+                <button type="button" class="btn-close btn-close-white" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <form id="add-form" action="/api/job/create" method="POST">
+                    <div class="mb-3">
+                        <label for="provider" class="form-label">Provider</label>
+                        <select class="form-select bg-white text-dark" id="provider" name="provider">
+                            <option selected disabled>Bitte auswählen</option>
```

Comment:
Remove german select option

Reply:
Fixed in [5575a06](https://github.com/Plaenkler/DDNS/pull/3/commits/5575a06b27508c0fae771c2a1a54206548c34498)